rails_event_store_active_record/Makefile

```diff
@@ -2,7 +2,8 @@ GEM_VERSION  = $(shell cat ../RES_VERSION)
 GEM_NAME     = rails_event_store_active_record
 REQUIRE      = $(GEM_NAME)
 IGNORE       = RailsEventStoreActiveRecord::IndexViolationDetector\#detect \
-               RailsEventStoreActiveRecord::PgLinearizedEventRepository*
+               RailsEventStoreActiveRecord::PgLinearizedEventRepository* \
+               RailsEventStoreActiveRecord::EventRepository#update_messages
```

Comment:
We could add a separate task for it, because when https://github.com/zdennis/activerecord-import/issues/540 will be done we would probably be able to do that

Reply:
👍